examples/point-cloud-laz/package.json

```diff
@@ -4,10 +4,14 @@
   "license": "MIT",
   "scripts": {
     "start": "webpack-dev-server --progress --hot --open",
-    "start-local": "webpack-dev-server --env.local --progress --hot --open"
+    "start-local": "webpack-dev-server --env.local --progress --hot --open",
+    "build-clean": "rm -rf dist && mkdir dist",
+    "build-script": "NODE_ENV=production webpack --env.prod=true",
+    "build-copy": "cp index.html dist",
+    "build": "npm run build-clean && npm run build-script && npm run build-copy"
   },
   "dependencies": {
-    "deck.gl": ">=4.2.0-alpha.9",
+    "deck.gl": "4.2.0-alpha.24",
```

Comment:
Is the version pinning intentional? If not use `>=4.2.0-alpha.24`

Reply:
I want to make the examples to be more stable to be published, that's why pinning the version to a specific one which is tested fully and works well. Same thing as the lock file .